Src/CSharpier.Tests/FormattingTests/TestFiles/Comments_EdgeCase.cst

```diff
@@ -21,6 +21,10 @@ public class ClassName
      */
     public class ClassName2 { } 
 
+    /* start
```

Comment:
So, with this case the start doesn't move.
Can you indent this line by another level?

I want to make sure that when it reduces indent of the first line, it doesn't try to make the second line go negative.

Reply:
Okay, I updated the test case myself.

Reply:
Nice catch!